src/RoutingApp.php

```diff
@@ -43,9 +40,12 @@ public function call(Context $context) {
 		// Force pages to their designated protocol if specified
 		if ($route->getProtocol() != null) {
 			if ($currentProtocol != $route->getProtocol()) {
-				$url = Url::fromString(sprintf("%s://%s/", $route->getProtocol(), $currentHost))
-						->join($currentUrl)
-						->addParams($_GET);
+				$getParams = implode("&", $_GET);
```

Comment:
$getParams extracted. I don't see them used?

Reply:
Whoops, left that in from testing to see how the GET params were being used and sent in. Remove it.